Source/Core/PixelFormat.js

```diff
@@ -144,6 +146,28 @@ define([
         /**
          * @private
          */
+        componentLength : function(pixelFormat) {
```

Comment:
This actually is for that case where `RGB` is three.

`PixelDatatype` figures out the sizeInBytes for each component, and the two numbers are multiplied for a final size.